adding_a_license_to_your_repository_v2.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: Adding-a-License-to-Your-Repository
+aspect: processes
+stages:
+  - “development”
+short_description: Every product should be covered by a license. It is your creative work, you own it and you should decide what other people can or cannot do with it.
+tags:
+  - license
+  - readme
+  - software documentation
+  - product
+  - documentation
+  - information
+  - description
+  - introduction
+  - installation
+  - writing
+keywords:
+  - license
+  - repository
+  - royalty terms
+  - copyright
+---
+
+# Adding a License to Your Repository

```

Comment:
I´m not sure about that long name. Could we call it just e.g. "License in Repository" or something similar?